diagrams: Fixed replace text for alter_table_set_storage_param

nickvigilante opened the pull request:
The diagram was causing the Publish SQL Grammar Diagrams to fail for release-22.1. This change should fix that.

Fixes DOC-5649

Release note: None

Release justification: SQL grammar diagram update

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/87878)
<!-- Reviewable:end -->

Commit:
diagrams: Fixed replace text for alter_table_set_storage_param
The diagram was causing the Publish SQL Grammar Diagrams to fail
for release-22.1. This change should fix that.

Fixes DOC-5649

Release note: None

Release justification: SQL grammar diagram update
